src/lib.rs

```diff
@@ -114,22 +113,24 @@ pub const DEFAULT_ENDIAN: Endian = Endian::Little;
 #[cfg(target_endian = "big")]
 pub const DEFAULT_ENDIAN: Endian = Endian::Big;
 
+type StaticLocation = &'static std::panic::Location<'static>;
+
 /// Potential values of a `Binding`.
 enum BindingValue {
     /// A known constant value.
     Constant(u64),
     /// Equal to some other binding plus a constant.
     From(Rc<Binding>, i64),
     /// Free to take on any value.
-    Unconstrained,
+    Unconstrained(StaticLocation),
 }
 
 impl fmt::Debug for BindingValue {
```

Comment:
I wonder if we could just `#[derive(Debug)]` for this instead? I have no idea why this is manually implemented TBH.